integration-tests/zombienet/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "zombienet-sdk-tests"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+subxt = { version = "0.35.0", default-features = false, features = ["native"] }
+tokio = "1.35.0"
+tracing-subscriber = "0.3.18"
+zombienet-sdk = { git = "https://github.com/paritytech/zombienet-sdk", tag = "v0.2.1" }
```

Comment:
Why we're not pointing to `crates.io`? I saw that 0.2.2 is already there

Reply:
Yes, I will change it to use crates.io 👍